src/main/resources/changesets/changelog-consent-93.0.xml

```diff
@@ -15,4 +15,4 @@
         ) 
       </sql>
   </changeSet>
-</databaseChangeLog>
\ No newline at end of file
+</databaseChangeLog>
```

Comment:
should we make sure this one isn't being changed just in case there is some sort of hashing error ?

Reply:
you're right, I didn't realize I made a change to this file when resolving the merge conflict with my branch. thank you!